web/models/meeting.ex

```diff
@@ -1,19 +1,54 @@
 defmodule Meetings.Meeting do
-  defstruct title: "", date: DateTime.utc_now(), duration: 0, location: "", description: "", email: "", agenda: ""
+  defstruct title: "", location: "", date: DateTime.utc_now(), duration: 0
 
+  alias Meetings.MeetingType
+  alias Meetings.MeetingDate
```

Comment:
If this had been set up in Phoenix from the get go, it might have been modelled differently, but it came out of the Python scraping, which was dumping to a MySQL database, where those three tables made sense.